lib/error.js

```diff
@@ -100,8 +109,8 @@ AppError.prototype.backtrace = function (traced) {
 }
 
 /*/
-  Translates an error from Hapi format to our format
-/*/
+ Translates an error from Hapi format to our format
```

Comment:
nit: do we need this diff here? also the `/*/` is strange, if we are changing this as part of this PR can we just use 
```
/** 

*/
```